views/extras.tmpl

```diff
@@ -0,0 +1,87 @@
+  <link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg">
+  <link rel="shortcut icon" href="/images/favicon/favicon.ico">
+  <meta name="apple-mobile-web-app-title" content="deCRED - decentralized credit">
+  <meta name="application-name" content="deCRED - decentralized credit">
+  <meta name="msapplication-TileColor" content="#2973ff">
+  <meta name="msapplication-TileImage" content="/images/favicon/mstile-144x144.png">
+  <meta name="msapplication-config" content="/images/favicon/browserconfig.xml">
+    <link rel="icon" href="/favicon.ico">
+    <title>dcrdata web</title>
+    <link href="/css/main.css" rel="stylesheet">
+    <link rel="stylesheet/less" type="text/css" href="/css/dynamic.less">
+    <!-- <script src="/js/Chart.min.js"></script> -->
+    <script src="/js/clipboard.min.js"></script>
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+    <style>
+        a {
+            color: white;
+            text-decoration: underline;
+        }
+        .copy_btn {
+            background-color: #2971FF;
+            border: none;
+            padding: 0 0;
+        }
+        .copy_btn img {
+            width: 10px;
+        }
+    </style>
+</head>
+{{end}}
+
+{{define "navbar"}}
+<script>
+        new Clipboard('.copy_btn');
+</script>
+<nav class="navbar navbar-default navbar-fixed-top">
+        <div class="container wrapper">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand" href="#">
+                    <svg class="svg-logo">
+                        <use xlink:href="/images/sprites.svg#svg-decred-logo" />
+                    </svg>
+                </a>
+            </div>
+            <div id="navbar" class="navbar-collapse collapse">
+                <ul class="nav navbar-nav">
+                </ul>
+                <ul class="nav navbar-nav navbar-right">
+                    <li><a href="https://github.com/dcrdata/dcrdata" title="dcrdata on GitHub" title="_blank">GitHub</a></li>
+                    <li><a href="https://github.com/dcrdata/dcrdata#json-rest-api" title="API Endpoints" title="_blank">API</a></li>
+                    <li><a href="/api/status" title="API Status" title="_blank">API Status</a></li>
+                    <li><a href="/explorer" title="Block Explorer" title="_blank">Explore</a></li>
```

Comment:
I discovered in some work on another branch that this as supposed to be `target="_blank"` instead of `title="_blank"`.  Might as well change them here.